src/jit/gentree.cpp

```diff
@@ -6246,15 +6246,310 @@ bool GenTree::TryGetUse(GenTree* def, GenTree*** use)
     assert(def != nullptr);
     assert(use != nullptr);
 
-    for (GenTree** useEdge : UseEdges())
+    switch (OperGet())
+    {
+        // Leaf nodes
```

Comment:
Let's discuss the utility and efficiency of operator kinds in #12443 and leave this as-is for now.